drivers/common/opentimers.c

```diff
@@ -358,7 +361,7 @@ void opentimers_timer_callback(void){
         if (opentimers_vars.timersBuf[i].isrunning==TRUE){
             // all timers in the past within TIMERTHRESHOLD ticks
             // (probably with low priority) will mared as Expired.
-            if (opentimers_vars.currentTimeout-opentimers_vars.timersBuf[i].currentCompareValue <= TIMERTHRESHOLD){
+            if ((PORT_TIMER_WIDTH)(opentimers_vars.currentTimeout-opentimers_vars.timersBuf[i].currentCompareValue) <= TIMERTHRESHOLD){
```

Comment:
I don't think this will make a difference with this changes. Both variable are PORT_TIMER_WIDTH type already.